dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java

```diff
@@ -0,0 +1,81 @@
+package io.dockstore.webservice.resources.proposedGA4GH;
+
+import io.dropwizard.hibernate.UnitOfWork;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.swagger.api.NotFoundException;
+import io.swagger.model.Tool;
+import org.apache.http.HttpStatus;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+@Path("/extendedGA4GH")
```

Comment:
Let's put these resources at "/api/ga4gh/v1/extended"

See https://github.com/ga4gh/dockstore/blob/develop/dockstore-webservice/src/main/java/io/swagger/api/ToolsApi.java#L37 for the variable you should use